rpmlb/cli.py

```diff
@@ -48,6 +51,11 @@
     type=click.Path(exists=True, dir_okay=False, resolve_path=True),
     help='Instructions for custom downloader and builder.',
 )
+@click.option(
+    '--dist', metavar='DIST',
```

Comment:
Suggestion: Replace the `DIST` metavar with the `DIST_RE` pattern (or similar), to inform user early what they may pass as a value:

```python
@click.option('--dist', metavar='(fc|el|centos)[0-9]*', ...)
```
```shell
$ rpmlb --help
...
--dist (fc|el|centos)[0-9]*    Specify a distribution...
...
```